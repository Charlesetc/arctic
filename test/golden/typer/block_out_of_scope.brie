define main [
        define f [ < x = [ 2 ] > ]
        (f ()).x ()
]
