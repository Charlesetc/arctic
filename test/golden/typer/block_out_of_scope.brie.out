root[(ident(define):: ident(main):: block[][(ident(define):: ident(f):: block[][(<x = (block[][(ident(2)::)::]::)::>::)::]::Function)::, ((ident(f)::Function ()::Unit)::<x = Function>.x::Function ()::Unit)::Integer]::Function)::]::